### PR TITLE
Upload the To-Do-list project 

### PR DESCRIPTION
**Pull request by MoRaad97**
### in this PR we add:
- Add the UI.
- Add the remove and add task functions.
- Add the edit task function.
- Add the clear all completed tasks function.
- Add the change completed state function.
- Add function to save all changes to local storage.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import './styles/all.min.css';
+import {
+  addTaskToArray, getTaskFromStore, deleteTaskWith, editTask, clearAllComplete,
+} from './add-remove.js';
+
+const input = document.querySelector('.ph');
+const submit = document.querySelector('.enter');
+const taskEvent = document.body.querySelector('.tasks-list');
+
+// get task from local storage on load
```
Comment (efoosag) [inline src/index.js:11]: Please check and remove this comment and other comments althrough your code, use space to separate block of code

File: src/complete.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+const addCheckedAttFromStore = (tasks) => {
+  for (let i = 0; i < tasks.length; i += 1) {
+    const checkedInputElements = document.getElementsByClassName('check-input');
+    if (tasks[i].completed) {
+      checkedInputElements[i].setAttribute('checked', '');
+      //   console.log(checkedInputElements[i])
+    } else {
+      checkedInputElements[i].removeAttribute('checked');
+      //    console.log(checkedInputElements[i])
```
Comment (efoosag) [inline src/complete.js]: Please remove console.log from your code to ensure javaScript best practice.

**Review by efoosag — CHANGES_REQUESTED** [review]
**You have done a very Good Job with your project** 👍 

- No linters error 👌 
- Html and CSS best practices 👌 
- DRY, KISS, and YAGNI 👌 

**However, I noticed alot of comments in your code, please remove to maintain JavaScript Best Practice.**

After That, You are good to go.

File: src/index.js (hunk 1 above)
Comment (MoRaad97) [reply src/index.js:11]: Hi Victor I think It's good to have comments to guide the reviewer.

**Review by efoosag — APPROVED** [review]
### Approved
**Very Good Job 👍 
Now your project is completed.**

**Pull request merged**